src/instana/instrumentation/aws/dynamodb.py

```diff
@@ -0,0 +1,40 @@
+# (c) Copyright IBM Corp. 2025
+
+from typing import TYPE_CHECKING, Any, Callable, Dict, Sequence, Type
+
+if TYPE_CHECKING:
+    from botocore.client import BaseClient
+
+from instana.log import logger
+from instana.singletons import tracer
+from instana.span_context import SpanContext
+
+try:
```

Comment:
You remove the try-except from the code because you are not importing the `boto3` (and no other package).

Reply:
Done